### PR TITLE
fix(DB/creature): Remove Bonelashers inside of a wall.

### PR DESCRIPTION
**Pull request by heyitsbench**
## Changes Proposed:
- Removes two [Bonelashers](https://www.wowhead.com/wotlk/npc=18470) from the walls of Auchindoun.

## Issues Addressed:

- Closes unreported issue.

## SOURCE:

It's simply illogical!

## Tests Performed:

- Untested.

## How to Test the Changes:

1. Run the query.
2. `.go xyz -3381.7 4616.8 -22.4 4.3`
3. Turn on nameplates.
4. Observe a lack of Bonelashers inside walls.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by heyitsbench** [comment]
There are more than one Bonelashers in the walls, gimme a bit.

**Comment by heyitsbench** [comment]
The birds have been exterminated.

**Review by IntelligentQuantum — APPROVED** [review]

**Comment by Samsequel** [comment]
@Gultask 

Should we remove these or are they just wrongly spawned (are they extra, or not :?)

**Comment by Gultask** [comment]
Impossible to know what happened to create those but this isn't the first time, specially in Outland.
Remove them. If we go back to re-sniff the spawns it'd be easier to just re-create it rather than checking/speculating the individual spawns of a wandering creature.

File: data/sql/updates/pending_db_world/bonelasher-oob.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+DELETE FROM `creature` WHERE `guid` IN (66458, 66459);
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/bonelasher-oob.sql:1]: Missing AND `id1`=XXXX

**Review by Gultask — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/bonelasher-oob.sql (hunk 1 above)
Comment (heyitsbench) [reply data/sql/updates/pending_db_world/bonelasher-oob.sql:1]: Update [the wiki](https://www.azerothcore.org/wiki/sql-standards) with that info, please.

Comment (Gultask) [reply data/sql/updates/pending_db_world/bonelasher-oob.sql]: No no I meant id1 😄 
There are hundreds of creatures with curhealth=5715, and thousands with map=530

```
DELETE FROM `creature` WHERE `guid` IN (66458, 66459) AND `id1`=18470;
```

Comment (heyitsbench) [reply data/sql/updates/pending_db_world/bonelasher-oob.sql]: And there are most definitely > 2 creatures with an id1 of 18470, I really don't see the point of specifying further than a keyed column where duplicates aren't possible.

Comment (Gultask) [reply data/sql/updates/pending_db_world/bonelasher-oob.sql]: It's been a longstanding practice for quite a while

image

Comment (heyitsbench) [reply data/sql/updates/pending_db_world/bonelasher-oob.sql]: > It's been a longstanding practice for quite a while

I would not consider under two months to be "quite a while". But even still, it's not in the standard operating procedure on the wiki with regards to SQL queries, so I still wouldn't call it a standard practice.

Comment (Gultask) [reply data/sql/updates/pending_db_world/bonelasher-oob.sql]: That's just an example I found. I've always been taught to add GUIDs and IDs together.

**Review by Gultask — APPROVED** [review]

File: data/sql/updates/pending_db_world/bonelasher-oob.sql (hunk 1 above)
Comment (heyitsbench) [reply data/sql/updates/pending_db_world/bonelasher-oob.sql]: Well for people like me, these standards need to be on the wiki. I can only scan through so many issues/PRs I have nothing to do with.

**Pull request merged**